src/client/Microsoft.Identity.Client/Cache/ICacheSessionManager.cs

```diff
@@ -26,7 +26,7 @@ internal interface ICacheSessionManager
         
         Task<IEnumerable<IAccount>> GetAccountsAsync();
         Task<IDictionary<string, TenantProfile>> GetTenantProfilesAsync(string homeAccountId);
-
+        Task<Account> GetAccountAssociatedWithAccessTokenAsync(MsalAccessTokenCacheItem msalAccessTokenCacheItem);
```

Comment:
Deleting `GetTenantProfilesAsync` making it private in TokenCache. Leaving `GetIdTokenCacheItemAsync` since I did not combine it with tenant profiles as mentioned in https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/3199#discussion_r821187217